### PR TITLE
Symbology for ch.bfe.bikesharing

### PR DESCRIPTION
**Pull request by nadine-piveteau**
Still wip

### CODE REVIEW

**Comment by loicgasser** [comment]
I like the test links.

**Comment by nadine-piveteau** [comment]
The legend is not yet ready. I will provide the testlink as soon as I have that. Or would you like me to provide you with an interim testlink?

**Comment by loicgasser** [comment]
It's fine. Style looks good, just mention what's missing in the description.

**Comment by nadine-piveteau** [comment]
Here we are, the [testlink](https://mf-chsdi3.int.bgdi.ch/ltpin_bikesharing/shorten/7776575c41).

File: production.ini.in [hunk 1]
```diff
@@ -21,7 +21,8 @@ pyramid.includes = pyramid_tm
 sqlalchemy.are.url = postgresql://${dbhost}:${dbport}/are_${dbstaging}
 sqlalchemy.bafu.url = postgresql://${dbhost}:${dbport}/bafu_${dbstaging}
 sqlalchemy.bak.url = postgresql://${dbhost}:${dbport}/bak_${dbstaging}
-sqlalchemy.bod.url = postgresql://${dbhost}:${dbport}/bod_${dbstaging}
+#sqlalchemy.bod.url = postgresql://${dbhost}:${dbport}/bod_${dbstaging}
+sqlalchemy.bod.url = postgresql://${dbhost}:${dbport}/bod_ltpin_master
```
Comment (loicgasser) [inline production.ini.in]: please revert

Comment (nadine-piveteau) [reply production.ini.in]: It's on my todo list before the deploy. I am still waiting for the final feedback of the Datenherren and after that I will change everything. Everything is: 
- revert changes in `production.ini.in`
- adapt paths from `json/test/*` to `json/*` in mapfile and json-stylesheet once the definitive data is moved to the correct folder.

**Comment by nadine-piveteau** [comment]
The layer should now be ready for the next deploy.

**Pull request merged**